app/services/message_sender.rb

```diff
@@ -15,12 +16,19 @@ def run
         client.chat_postMessage(
           channel: channel_id,
           as_user: true,
-          text: message.body
+          text: escaped_message_body
         )
       end
     end
   end
 
+  def escape_specific_characters(message_body)
```

Comment:
since we're only using this method inside this class, I think it can be `private`